careers.md

```diff
@@ -1,52 +1,32 @@
 ---
 layout: technical
-title: Carreers at Phoxygen
+title: Careers at Phoxygen
 ---
 
-# Phoxygen is hiring
+# Phoxygen is hiring!
 
-## Senior Javascript Developer at Phoxygen
+## Senior JavaScript Developer
 
-The technical team at Phoxygen is looking for experienced Javascript developer to work on Mozilla technologies. You'll be fully part of the team that work on Firefox OS
+Phoxygen is looking for an experienced JavaScript developer to work on Mozilla technologies. You’ll be fully part of the team that works on Firefox OS.
 
-### A few informations regarding your mission:
+### Mission
 
-- Translate UX design and wireframe in high quality code
-- Write and maintain performant, secured and reliable code
-- Make sure to understand and write you code with a sole objective, performance and reactivity of the frontend side
-- Know how to get along with the team and work in an efficient way remotely
-- Identify and fix bugs
-- Work in a peer-review fashion
-- Leark on workflow like github and bugzilla to fit in the company
+- Develop progressive web applications with optimal efficiency and performance
+- Translate UX designs and wireframes into high-quality code
+- Work remotely with a distributed team of passionate developers
 
-## Your qualification
+### Qualifications
 
-- Engineer diploma or equivalent professional experience.
-- More than 3 year as a software developer
-- Skilfully written and spoken english, mastering french is a plus, but not mandatory. Other mastered languages welcome.
+- Your open-source repositories are your best recommendation!
+- Engineering degree or equivalent professional experience
+- Successful 3+ year experience as a software developer
```

Comment:
http://grammar.ccc.commnet.edu/GRAMMAR/compounds.htm seems to be a good source of info about compound names. This is the first time I heard about compound names so that was good reading :)

It seems `three-year-old` should be hyphenated. But since there are compound names without hyphens both spelling are probably allowed.
In the case of `3+ year`, I think I would agree that it's a compound name and thus doesn't take a plural ending. I would be more certain with `3-year` and I think my hesitation comes from the `+`.